### PR TITLE
Hover titles 500

### PR DESCRIPTION
**Pull request by wgilling**
_pending approval of the text used_

This feature branch adjusts the buttons that are rendered in several places in order to add a title="some title" attribute to them. Since some of these are in twig files and the rest are adjustments to either the block build() routine or finally changed in a form alter hook, you should be able to see these changes after checking out the branch and clearing the cache.

##### "Copy citation" button:

- To see the adjusted button text, navigate to an item overview page or collection that contains a citation and inspect the button html or hover until the title appears and confirm title = "Copy citation".

##### "Download {type}" button:

- To see the adjusted button text, navigate to an item overview page that has attached media and inspect the various download button options to confirm title = "Download {type}" where {type} is either _original_ or _derivative_.

##### ASU Search button (top nav bar):

- inspect button to see the title = "Search ASU"

##### Search results - refine search button:

- navigate to any site search or collection search results (and optionally even facet the results) and confirm that either button title = "Search" or "Search collection" as appropriate

##### Feedback button:

- navigate to any page that has the Feedback button and confirm that the title = "Feedback"

### CODE REVIEW

File: web/themes/custom/asulib_barrio/templates/field/field--node--field-preferred-citation--asu-repository-item.html.twig [hunk 1]
```diff
@@ -57,7 +57,7 @@
   <div{{ attributes.addClass(classes, 'field__item') }}>
     {#<strong>{{ nodetitle }}</strong>#}
     <p>{{ 'This is a suggested citation. Consult the appropriate style guide for specific citation guidelines'|t }}.</p>
-    <p id="citation-text"><strong>{{ item.content }}</strong></p>
-    <button id="copy_citation" class="btn btn-primary btn-md copy_button">Copy citation</button>
+    <p id="citation-text"><strong>{{ item.content|render|striptags }}</strong></p>
```
Comment (elizoller) [inline web/themes/custom/asulib_barrio/templates/field/field--node--field-preferred-citation--asu-repository-item.html.twig]: what's the purpose of the `|render|striptags` here? i believe the citation text can contain html for bold, italics, etc - so wouldn't stripping remove that?

**Comment by elizoller** [comment]
Just the one comment/question then I think this is good to go

File: web/themes/custom/asulib_barrio/templates/field/field--node--field-preferred-citation--asu-repository-item.html.twig (hunk 1 above)
Comment (wgilling) [reply web/themes/custom/asulib_barrio/templates/field/field--node--field-preferred-citation--asu-repository-item.html.twig]: without this, the multiple lines from any citation value would be separated into separate <p> tags ultimately which caused the javascript to not be able to copy any value into the clipboard when clicking the button. For example, the HTML was coming out like this and the javascript was only getting what was in the p#citation-text element:
```
  <p>This is a suggested citation. Consult the appropriate style guide for specific citation guidelines. </p>
  <p id="citation-text"><strong></strong></p>
  <p><strong>The related digital object has blah blah -- this is the citation for that.</strong></p>
  <p></p>
```

Comment (wgilling) [reply web/themes/custom/asulib_barrio/templates/field/field--node--field-preferred-citation--asu-repository-item.html.twig]: I worked with Debbie on this issue and she said that she has had to add |render|striptags in a lot of places due to how drupal twigs would sometimes get populated.

Comment (elizoller) [reply web/themes/custom/asulib_barrio/templates/field/field--node--field-preferred-citation--asu-repository-item.html.twig]: this field is rich html so we can't just strip tags. this is not related to this issue - it also shouldn't be wrapped in a `<strong>` or that'll mess with the potential formatting that a user supplies. the `p` tag should be changed to a `div` or some other html element that can have `p` children and then it should work fine

Comment (elizoller) [reply web/themes/custom/asulib_barrio/templates/field/field--node--field-preferred-citation--asu-repository-item.html.twig]: i changed it with https://github.com/asulibraries/islandora-repo/pull/524/commits/d60571cc7667fd1462424045e3eccc1157f76626

Comment (wgilling) [reply web/themes/custom/asulib_barrio/templates/field/field--node--field-preferred-citation--asu-repository-item.html.twig]: thank you --- in my test item, I did not use any formatting and I overlooked how the striptags would impact this... but I suspect that the user will get any markup in their copied text but I suppose that is completely the idea.

**Pull request merged**